### DISCUSSION TITLE
How to stop the server?

### DISCUSSION THREAD

**Discussion by DrSensor** [lumeland/lume#296] [Q&A] [answered]
I try to write e2e test for my site which use [On Demand](https://lume.land/plugins/on_demand/) plugin and just notice `Server` class doesn't have `.stop()` method 😅

Any reason why not just make `class Server extends std.HTTPServer`?

**Comment by oscarotero** [comment]
Good idea. Never thought about that :)

**Comment by oscarotero** [comment, marked as answer]
I just commited a change that implements `.stop()` to the server. You can test it after upgrading Lume to the latest *development* version with `deno task lume upgrade --dev`.
